app/assets/javascripts/admin/controllers/admin-user-index.js.es6

```diff
@@ -41,6 +41,16 @@ export default Ember.Controller.extend(CanCheckEmails, {
     return userPath(`${username}/preferences`);
   },
 
+  @computed('model.second_factor_enabled')
+  show_second_factor(second_factor_enabled) {
```

Comment:
In Javascript we use camelCase style for object properties. This should be `showSecondFactor` and `displaySecondFactorEnabled`.

Having said that, `showSecondFactor` is basically an alias to a model property.  It wouldn't be bad to just have `{{#if model.second_factor_enabled}}` in your template.